packages/lesswrong/lib/collections/revisions/views.ts

```diff
@@ -8,9 +8,21 @@ declare global {
     fieldName?: string
     before?: Date|string|null,
     after?: Date|string|null,
+    userId?: string
   }
 }
 
+Revisions.addView('revisionsByUser', (terms: RevisionsViewTerms) => {
+  return {
+    selector: {
+      userId: terms.userId,
+      // tag: {$ne:null} for some reason this returns a view where tag is often null. if we filter by this then it returns nothing.
```

Comment:
The thing that's happening here is that `tag` is a resolver-only field. Those fields only exist when you ask our API for them, and are not stored in the database.

https://github.com/centre-for-effective-altruism/EAForum/blob/9f2d16e1767bc7d24d3e12d5d32bebb32a310a5d/packages/lesswrong/lib/collections/revisions/schema.ts

Reply:
I changed the query to specify `{collectionName: 'Tags'}` and everything worked.